src/FilesystemStore.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+const fs = require('fs').promises;
+const { join } = require('path');
+const jsonl = require('./jsonl');
```

Comment:
Why do you need `jsonl` folder with single file, it would be better to move `jsonl/index.js` to `jsonl.js` and import as:
```suggestion
const jsonl = require('./jsonl.js');
```